src/components/molecules/Visualizer/storybook.tsx

```diff
@@ -77,6 +77,8 @@ export const context: ProviderProps = {
   zoomIn: act("zoomIn"),
   zoomOut: act("zoomOut"),
   overrideLayerProperty: act("overrideLayerProperty"),
+  layersInViewport: act("layersInViewport"),
+  viewport: act("viewport"),
```

Comment:
```suggestion
  layersInViewport: [],
  viewport: undefined,
```

Reply:
`layersInViewport` and `viewport` are still functions, so I need to use act, instead of assigning a direct value.
What do you think?

[src/components/molecules/Visualizer/Plugin/context.tsx](https://github.com/reearth/reearth-web/blob/c06772a2036e0eba472e2fdb56e43f060c6d5cf8/src/components/molecules/Visualizer/Plugin/context.tsx#L55-L56)

Reply:
Oh ok you can use act